components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/DateRangePicker.jsx

```diff
@@ -175,13 +177,19 @@ export default class DateRangePicker extends Widget {
         if (availableGranularities.indexOf(this.capitalizeCaseFirstChar(granularity)) > -1) {
             return granularity;
         } else {
-            return this.lowerCaseFirstChar(availableGranularities[0]);
+            return availableGranularities[0].toLowerCase;
```

Comment:
`toLowercase()` is a function, not a property 
https://www.w3schools.com/jsref/jsref_tolowercase.asp

Reply:
Fixed and merged in PR #63